docs/en/stack/ml/df-analytics/flightdata-classification.asciidoc

```diff
@@ -0,0 +1,484 @@
++
+--
+
+[source,console]
+--------------------------------------------------
+GET _ml/data_frame/analytics/model-delayed-flights/_stats
+--------------------------------------------------
+// TEST[skip:TBD]
+
+
+The API call returns the following response: 
+
+[source,console-result]
+----  
+{
+  "count" : 1,
+  "data_frame_analytics" : [
+    {
+      "id" : "model-delayed-flights",
+      "state" : "stopped",
+      "progress" : [
+        {
+          "phase" : "reindexing",
+          "progress_percent" : 100
+        },
+        {
+          "phase" : "loading_data",
+          "progress_percent" : 100
+        },
+        {
+          "phase" : "analyzing",
+          "progress_percent" : 100
+        },
+        {
+          "phase" : "writing_results",
+          "progress_percent" : 100
+        }
+      ]
+    }
+  ]
+}
+----  
+
+The job has four phases. When all the phases have completed, the job 
+state becomes `stopped` and the results are ready to view and evaluate.
+--
+
+
+[[flightdata-classification-results]]
+==== Viewing results
+
+. Use the standard {es} search command to view the results in the destination 
+index:
++
+--
+
+[source,console]
+--------------------------------------------------
+GET df-flight-results/_search
+--------------------------------------------------
+// TEST[skip:TBD]
+
+
+The snippet below shows a part of a document with the annotated results:
+
+[source,console-result]
+----  
+          ...
+          "FlightDelay" : false, <1>
+          "DistanceMiles" : 8865.200239823978,
+          "FlightTimeMin" : 792.6198230421824,
+          "OriginWeather" : "Cloudy",
+          "dayOfWeek" : 1,
+          "AvgTicketPrice" : 816.1058877497752,
+          "Carrier" : "Kibana Airlines",
+          "FlightDelayMin" : 0,
+          "OriginRegion" : "CO-CUN",
+          "FlightDelayType" : "No Delay",
+          "DestAirportID" : "NRT",
+          "timestamp" : "2019-11-12T17:06:35",
+          "Dest" : "Narita International Airport",
+          "FlightTimeHour" : 13.210330384036373,
+          "Cancelled" : false,
+          "DistanceKilometers" : 14267.156814759282,
+          "OriginCityName" : "Bogota",
+          "DestWeather" : "Sunny",
+          "OriginCountry" : "CO",
+          "ml__id_copy" : "-7OQg24B95sXiIrm0EZo",
+          "DestCountry" : "JP",
+          "DestRegion" : "SE-BD",
+          "OriginAirportID" : "BOG",
+          "DestCityName" : "Tokyo",
+          "ml" : {
+            "top_classes" : [ <2>
+              {
+                "class_probability" : 0.939335365058496, <3>
+                "class_name" : "false" <4>
+              },
+              {
+                "class_probability" : 0.06066463494150393,
+                "class_name" : "true"
+              }
+            ],
+            "FlightDelay_prediction" : "false", <5>
+            "is_training" : false <6>
+          }
+----
+<1> The `dependent_variable` with the ground truth value. This is what we are 
```

Comment:
These callouts render as small roman numbers (`i`, `ii`, etc.) but in the json snippet above they are  arabic numbers (`<1>`, `<2>`, etc.). This may make it harder for the user to match.